_includes/contact.html

```diff
@@ -4,26 +4,49 @@
   </div>
   <div class="uk-width-1-3">
     <h2>Contact Us</h2>
-    <form class="uk-form" action="{{ site.contact_form.host }}/contact_form" method="POST">
-      <input type="hidden" name="redirect_to" value="">
-      <input type="hidden" name="membership[list_id]" value="{{ site.contact_form.list_id }}">
-
-      <div class="uk-form-row">
-        <input class="uk-width-1-1" type="text" name="membership[person_attributes][email]" placeholder="Email" required>
-      </div>
-      <div class="uk-form-row">
-        <input class="uk-width-1-1" type="text" name="membership[person_attributes][name]" placeholder="Name" required>
-      </div>
-      <div class="uk-form-row">
-        <textarea class="uk-width-1-1" name="membership[notes_attributes][][content]" rows="10">
-        </textarea>
+    <form class="uk-form ajax" action="{{ site.contact_form.host }}/contact_form" method="POST">
+      <div class="uk-alert uk-alert-large uk-alert-success uk-hidden">
+        <h3>Thanks for contacting us.</h3>
+        <p>We'll get back to you as soon as possible</p>
       </div>
-      <div class="uk-form-row">
-        <input type="submit" class="uk-button uk-button-primary">
+      <div class="uk-alert uk-alert-danger uk-hidden"></div>
+      <div class="form-content">
+        <input type="hidden" name="membership[list_id]" value="{{ site.contact_form.list_id }}">
+
+        <div class="uk-form-row">
+          <input class="uk-width-1-1" type="text" name="membership[person_attributes][email]" placeholder="Email" required>
+        </div>
+        <div class="uk-form-row">
+          <input class="uk-width-1-1" type="text" name="membership[person_attributes][name]" placeholder="Name" required>
+        </div>
+        <div class="uk-form-row">
+          <textarea class="uk-width-1-1" name="membership[notes_attributes][][content]" rows="10">
+          </textarea>
+        </div>
+        <div class="uk-form-row">
+          <input type="submit" class="uk-button uk-button-primary">
+        </div>
       </div>
     </form>
   </div>
 </div>
 <script>
-  $(function() { $('[name=redirect_to]').val(document.location.protocol + '//' + document.location.host + '/thank_you.html'); })
+  $(function() { 
+    $('form.ajax').on('submit', function(e){
+      e.preventDefault();
+      $form = $(this);
+      $form.find('.uk-alert-danger').addClass('uk-hidden');
+
+      $.post($form.attr('action'), $form.serialize())
+        .done(function(data, textStatus, jqXHR){
+          $form.height($form.height());
+          $form.find('.form-content').addClass('uk-hidden');
+          $form.find('.uk-alert-success').removeClass('uk-hidden');
+        }).fail(function(jqXHR, textStatus, errorThrown) {
+          $form.find('.uk-alert-danger')
+            .html(textStatus + ": " + errorThrown)
```

Comment:
This will likely not function as intended.  Please use jqXHR.responseJSON.error and fallback to this;  properly written, back end will return errors as error or errors: [{}];  I can accommodate either.